.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,115 @@
+name: End to End Test
+
+on:
+  issue_comment:
+    types: [created]
+
+env:
+  UPTEST_AZURE_CREDS: ${{ secrets.UPTEST_AZURE_CREDS }}
+
+jobs:
+  check-triggered:
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/test-e2e') }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Checkout PR
+        id: checkout-pr
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh pr checkout ${{ github.event.issue.number }}
+          git submodule update --init --recursive
+          OUTPUT=$(git log -1 --format='%H')
+          echo "::set-output name=commit-sha::$OUTPUT"
```

Comment:
Suggestion similar to https://github.com/upbound/platform-ref-aws/pull/82/files#r998103618 . Can be tracked separately

Reply:
Yes, I think it is a good one to tackle as part of https://github.com/upbound/uptest/issues/31. Left a comment there.